src/devtools/client/debugger/src/actions/pause/paused.js

```diff
@@ -12,17 +12,28 @@ import { trackEvent } from "ui/utils/telemetry";
 // How many times to fetch an async set of parent frames.
 const MaxAsyncFrames = 5;
 
-export function fetchFrames(cx) {
+function failedToFetchFrames() {
+  return { type: "FAILED_TO_FETCH_FRAMES", pauseId };
+}
+
+function failedToCreatePause(executionPoint, pauseId) {
+  return { type: "FAILED_TO_CREATE_PAUSE", executionPoint, pauseId };
+}
+
+export function fetchFrames(pause) {
   return async function (dispatch, getState, { client }) {
     let frames;
     try {
-      frames = await client.getFrames();
-    } catch (e) {}
-    dispatch({ type: "FETCHED_FRAMES", frames, cx });
+      frames = await client.getFrames(pause);
+      dispatch({ type: "FETCHED_FRAMES", frames, pauseId: pause.id });
+    } catch (e) {
+      console.error({ e });
```

Comment:
:hand: do we want to leave the `console.error` in?

Reply:
I kind of like it personally.